src/tests/multihost/sssd/testlib/common/qe_class.py

```diff
@@ -191,6 +191,18 @@ def dnf_uninstall(self, package):
                                raiseonerr=False)
         return cmd.returncode
 
+    @property    
+    def detect_files_provider(self):
+        """
+        Detect the SSSD's files provider feature 
+        :param: None
+        :return str: True if we get the libsss_files.so file
```

Comment:
Fix 'trype' please.
Otherwise looks good to me.

Reply:
Updated.